build.properties

```diff
@@ -25,7 +25,7 @@ mo_base_name=MatterOverdrive
 mc_version=1.10.2
 mcp_mappings=snapshot_20160719
 mod_version=0.5.0
-forge_version = 12.18.1.2026
+forge_version = 12.18.3.2239
```

Comment:
by updating forge the bounding block error i was trying to fix and had fixed manually before fixed itself.